### ISSUE TITLE
Not an issue, I just wanted to say how much I'm loving the game!!

### ISSUE THREAD

**Issue by Ning1253** [austin-j-taylor/Invested#3]
The physics are really well balanced and allow for fluid gameplay, which is really fun. I'm getting the hang of using pushes, coins and pulls together to get to further away places, and I have currently finished everything except the two Luthadel rooms!

The story was really cool (if a tiny bit short but that made it also much more enjoyable to go through in one playthrough) and the sandbox challenge rooms are always intriguing and, well, a nice challenge. 

Basically, thank you very much for taking the time to make this game, it feels amazing, and I felt just like I was a Mistborn during the entire game!

Once again, thank you very much!

**Comment by artemos-git** [comment]
Glad to hear you like it! I'm having a lot of free time this semester so I'm continuing to have fun working on it, but I know it's going to be ages before my next major release.

Here's a preview of what I'm working on now. Still a lot of work to do, but... :)
https://gfycat.com/saltythirstyarawana

**Comment by Ning1253** [comment]
WOW That was a lot to take in, it looks amazing!!!!!!!! From the looks of it, a completely differerent gameplay style, and the animations look SLICK! Looking forwards to this for sure!
